pom.xml

```diff
@@ -548,12 +548,13 @@ under the License.
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.22.1</version>
         <configuration>
+          <parallel>classes</parallel>
```

Comment:
Do you know why apache/flink does not use parallel=classes? It might be safer to understand Flink's approach to make sure it is safe to use parallelize tests across classes before making this change.

Reply:
I have not found discussions about using this parameter in Flink, and I agree with it that running tests in parallel might bring risks. I'll remove this parameter.